### PR TITLE
Add Template Changes

### PR DESCRIPTION
**Pull request by chaitrali-r**
Added A4 Template and removed Skill and Work templates.

Also changed UI - Create VC template page

### CODE REVIEW

File: src/app/admin/add-template/add-template.component.ts [hunk 1]
```diff
@@ -98,35 +98,78 @@ export class AddTemplateComponent implements OnInit {
...
+          <tr>
+  
+            <td>
+              <div class="line">
+                <div>Certified to</div>
+                <div>
+                  <b>{{record.name}}</b>
+                </div>
+                <div>issued on</div>
+                <div style="padding: 1em;font-family: 'Open Sans', sans-serif">
```
Comment (Pratikshakhandagale) [inline src/app/admin/add-template/add-template.component.ts:149]: @chaitrali-r remove inline css

File: src/app/admin/add-template/add-template.component.ts [hunk 2]
```diff
@@ -98,35 +98,78 @@ export class AddTemplateComponent implements OnInit {
...
+                  {{ credential.issuancedate}}
+                   
+                </div>
+                <div>issued by </div>
+                <div>{{issuer.name}}</div>
+              </div>
+            </td>
+                  <td valign="bottom">
+              <img style="float:left;padding-left: 2em;" src="{{qrCode}}" alt="qr_code" />
```
Comment (Pratikshakhandagale) [inline src/app/admin/add-template/add-template.component.ts:158]: @chaitrali-r remove inline css

File: src/app/admin/create-entity/create-entity.component.ts [hunk 3]
```diff
@@ -1220,11 +1220,13 @@ export class CreateEntityComponent implements OnInit {
           if (i == (this.usecaseSchema.length - 1) && !errArr.length) {
             this.saveData();
             this.nextStep();
+            // alert('12');
           }
         }, (err) => {
           errArr.push(this.usecaseSchema[i].title);
 
           if (errArr.length == 1 && (errArr.includes('Common') || errArr.includes('common'))) {
+            alert('11');
```
Comment (Pratikshakhandagale) [inline src/app/admin/create-entity/create-entity.component.ts:1229]: @chaitrali-r remove alert()

Comment (chaitrali-r) [reply src/app/admin/create-entity/create-entity.component.ts:1229]: yes will remove alert

File: src/app/admin/add-template/add-template.component.ts [hunk 4]
```diff
@@ -98,35 +98,78 @@ export class AddTemplateComponent implements OnInit {
...
+      <style>
+        .line {
+          text-align: center;
+          font-family:'Lato';
+          font-size: 2em;
+        }
+      </style>
+    </head>
+    <body style="width: 900px">
+      <div style="margin: 1em;border-radius: 5px;background-color: white;height: 600px;border: 3px solid #B275FF;">
```
[the remark this answers is not in the document]
Comment (chaitrali-r) [reply src/app/admin/add-template/add-template.component.ts:137]: We need it for userHtml variable.

**Pull request merged**